airflow/config_templates/config.yml

```diff
@@ -1943,6 +1943,22 @@
       type: string
       example: ~
       default: "30"
+    - name: enable_health_check
```

Comment:
Can you add docs to [Logging & Monitoring](https://airflow.apache.org/docs/apache-airflow/stable/logging-monitoring/index.html) about this feature?